style.css

```diff
@@ -0,0 +1,127 @@
+.topnav{
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  background: white;
+  box-shadow: 0 3px 3px rgba(238, 238, 238, 0.6);
+}
+.nav-item-left{
+  width: 25%;
+  height: 70px;
+  padding-left: 15px;
+}
+.video-title{
+  background-color:white;
+}
+.video-description{
+  background-color: white;
+}
+
+.comment{
+  background-color: white;
+}
+
+.video-details{
+  background-color: white;
+}
+.view-count{
+  display: block;
+  width: 20%;
+  color: grey;
+  font-weight: bold;
+  margin-left: auto;
+  margin-right: 0;
+}
+
+
+.img-fluid {
+  max-height:100%;
+  height:auto;
+}
+
+.thumb-img {
+  max-width:100%;
+  width:auto;
+}
+
+$primaryColor: #13e293;
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.left-column{
+    width: 560px;
+    display: inline-block;
+}
+
+.right-column{
+  float: right;
+  background-color: white;
+  width: 24%;
+}
+
+body {
+  font-family: sans-serif;
+  background-color: rgb(241, 241 ,241);
+  color: rgb(24, 24, 24);
+}
+
+h1,
+h2,
+h3,
+h4,
+h5 {
+  margin: 0;
+  padding: 0;
+}
+
+.content {
```

Comment:
nested selectors will be used in your SASS project. For now, stick to your CSS 😔